operator-pipeline-images/operatorcert/entrypoints/set_cert_project_repository.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script for setting the cert project status
+"""
+import argparse
+import logging
+from datetime import datetime, timezone
+from typing import Any
+from urllib.parse import urljoin
+
+from operatorcert import pyxis
+from operatorcert.logger import setup_logger
+from operatorcert.utils import store_results
+
+LOGGER = logging.getLogger("set-cert-project-repository")
```

Comment:
The common logger for all scripts is called `operator-cert`. Please rename it.

Reply:
Done